src/chi/components/footer/footer.scss

```diff
@@ -0,0 +1,143 @@
+@import 'mixins';
+@import 'variables';
+
+.chi {
+  .o-footer {
+    align-items: center;
```

Comment:
@davidojedacl please remove all properties except background-color, width and z-index. As footer is a block element.